tests/dummy/app/controllers/snippet-edit-sample.ts

```diff
@@ -290,7 +288,7 @@ export default class RegulatoryStatementSampleController extends Controller {
           'https://publicatie.gelinkt-notuleren.vlaanderen.be/id/adres/',
       },
       snippet: {
-        endpoint: 'https://dev.reglementairebijlagen.lblod.info/sparql',
+        endpoint: 'https://dev.reglementairebijlagen.lblod.info/raw-sparql',
```

Comment:
why are we changing this to the uncached endpoint?

Reply:
Probably a mistake on my part :grimacing:

Reply:
Oh, no. It was me. I spotted that the other two sample pages used the uncached one, so I made this the same. It's true we should really go the other way and make them all cached.